drivers/hmis/lib/form_data/tarrant_county/ce_referral_steps/ce_shelter_referral.json

```diff
@@ -155,20 +156,80 @@
               }
             ]
           },
+          {
+            "text": "Have you stayed at a family shelter in the last three years?",
+            "type": "CHOICE",
+            "link_id": "stayed_at_family_shelter_last_three_years",
+            "mapping": {
+              "custom_field_key": "ce_shelter_referral_stayed_at_family_shelter_last_three_years"
+            },
+            "required": true,
+            "read_only": false,
+            "warn_if_empty": false,
+            "disabled_display": "HIDDEN",
+            "enable_behavior": "ANY",
+            "enable_when": [
+              {
+                "operator": "EQUAL",
+                "question": "family_or_individual",
+                "answer_code": "Family"
+              }
+            ],
+            "pick_list_options": [
+              {
+                "code": "Yes"
+              },
+              {
+                "code": "No"
+              }
+            ]
+          },
+          {
+            "text": "Which shelter(s)",
+            "type": "TEXT",
+            "link_id": "which_shelters",
+            "mapping": {
+              "custom_field_key": "ce_shelter_referral_which_shelters"
+            },
+            "required": true,
+            "read_only": false,
+            "warn_if_empty": false,
+            "disabled_display": "HIDDEN",
+            "enable_when": [
+              {
+                "operator": "EQUAL",
+                "question": "stayed_at_family_shelter_last_three_years",
+                "answer_code": "Yes"
+              }
+            ],
+            "enable_behavior": "ANY"
+          },
+          {
+            "text": "Why did you leave?",
+            "type": "TEXT",
+            "link_id": "why_did_you_leave",
+            "mapping": {
+              "custom_field_key": "ce_shelter_referral_why_did_you_leave"
+            },
+            "required": true,
+            "read_only": false,
+            "warn_if_empty": false,
+            "disabled_display": "HIDDEN",
+            "enable_when": [
+              {
+                "operator": "EQUAL",
+                "question": "stayed_at_family_shelter_last_three_years",
+                "answer_code": "Yes"
+              }
+            ],
+            "enable_behavior": "ANY"
+          },
           {
             "item": [
               {
                 "text": "<span style=\"font-size: 16px;\"><b>Adults</b></span>",
                 "type": "DISPLAY",
                 "link_id": "adults",
-                "enable_when": [
-                  {
-                    "operator": "EQUAL",
-                    "question": "family_or_individual",
-                    "answer_code": "Family"
-                  }
-                ],
-                "enable_behavior": "ALL",
```

Comment:
No change in behavior, just moved this check to the parent group item, rather than having each child check it.